### ISSUE TITLE
Question regarding groovy and Nexus 3.21.2

### ISSUE THREAD

**Issue by thorsteinbn** [ansible-ThoTeam/nexus3-oss#265]
Hi
I'm wondering, with the latest release of Nexus3, the following is found in the change log.

```
Disabled Groovy Scripting By Default

In order to make NXRM more secure, we have disabled Groovy scripting engine by default. This affects Groovy scripts as used through the REST API and through scheduled tasks.

```
More info: https://issues.sonatype.org/browse/NEXUS-23205

My question is, will this affect the role as it stands now?

**Comment by zeitounator** [comment]
1. I'm going to enable groovy in nexus 3,21.2 by default so that the role continues to work for now
2. I will have to accelerate the transition from groovy to REST API so that I can leave groovy disabled by default in the future.

**Comment by zeitounator** [comment]
Fix going through CI in #267.
